tests/data/loaders/test_load_predictions.py

```diff
@@ -213,4 +214,57 @@ def test_assumed_types_avoids_pandasFloat(self, input_series):
         pdt.assert_frame_equal(actual, expected)
 
 
+class TestDictionaryTypes:
+    @pytest.mark.parametrize(
+        "defined_types,expected_dtypes",
+        [
+            pytest.param(
+                [{"name": "downcast", "dtype": "float16"}],
+                {"downcast": "float16", "assumedTime": "datetime64[ns]", "keep": "int64"},
+                id="basic types",
+            )
```

Comment:
Since we went through all the work to add a parametrization, can we have a second example? 

This one test down casing to float16, can we try to downcast to int as well?

Reply:
added my "hardint" case of string(1.0) to int(1s